konfera/tests/test_models.py

```diff
@@ -135,21 +133,18 @@ def test_left_to_pay2(self):
 
 
 class ReceiptTest(TestCase):
-
     def test_string_representation(self):
         entry = models.Receipt(title="Test Receipt title")
         self.assertEqual(str(entry), entry.title)
 
 
 class RoomModelTest(TestCase):
-
```

Comment:
Those blank lines after `class` definition where there for readability purpose. I think we use it that way everywhere so let's keep them here as well (for consistency).